tutorial_fMRI.jl

```diff
@@ -0,0 +1,152 @@
+using LinearAlgebra
+using MKL
+using FFTW
+using ToeplitzMatrices
+using ExponentialUtilities
+using ForwardDiff
+using OrderedCollections
+using MetaGraphs
+using Graphs
+using ModelingToolkit
+using DataFrames
+using MAT
+
+#=
+TODO:
+    - explore adaptive connection strength between neural activity and hemodynamic response
+    - are there alternatives to small perturbations of initial conditions to introduce numerical stability?
+=#
+
+include("src/utils/typedefinitions.jl")
+include("src/VariationalBayes_MTKAD.jl")
+include("src/utils/mar.jl")
+include("src/models/neuraldynamics_symbolic.jl")
+include("src/models/measurement_symbolic.jl")
+include("src/utils/MTK_utilities.jl")
```

Comment:
For the final tutorial we should remove `include` statement to internal files and only use exported functions to not confuse the users and show how our public API works, so something like `using Neuroblox` or `using Neuroblox.SpectralDCM` should be enough.